bokeh/models/tests/test_annotations.py

```diff
@@ -119,6 +119,37 @@ def test_BoxAnnotation():
         "level",
     ], LINE, FILL)
 
+def test_Label():
+    label = Label()
+    assert label.plot is None
+    assert label.x is None
+    assert label.x_offset is None
+    assert label.x_units == 'data'
+    assert label.y is None
+    assert label.y_offset is None
+    assert label.y_units == 'data'
+    assert label.angle == 0
+    assert label.angle_units == 'rad'
+    assert label.x_range_name == 'default'
+    assert label.y_range_name == 'default'
+    assert label.level == 'overlay'
+    yield check_label, label
```

Comment:
can i suggest that with the addition of `label`, it would be helpful to rename the method `check_label` to, at least, `check_label_text` but perhaps more specifically `check_label_text_properties_defaults`